templates/baseline/baseline.jinja2

```diff
@@ -1,3 +1,32 @@
+- test:
+    timeout:
+      minutes: 10
+    definitions:
+    - repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: baseline
+          description: "baseline test plan"
+          os:
+            - debian
+          scope:
+            - functional
+          environment:
+            - lava-test-shell
+        run:
+          steps:
+            - dmesg --level=warn --notime -x -k > dmesg.warn
+            - >
+                for level in err crit alert emerg; do
+                  dmesg --level=$level --notime -x -k > dmesg.$level
+                  test -s dmesg.$level && res=fail || res=pass
+                  lava-test-case $level --result $res
```

Comment:
Would be useful to have an count here as well for each level (probably just a `cat dmesg.$level |wc -l)
We could report that with the `--measurement` option to lava-test-case.  Not sure what to use for `--units` though.  Maybe just "messages"?

Reply:
Related, the warnings could always be reported as a pass, but using `--measurement` you could at least report a count of warnings.  Could be useful for tracking increasing/decreasing warning count.

Reply:
c.f. https://docs.lavasoftware.org/lava/writing-tests.html#recording-test-case-measurements-and-units

Reply:
True, counting the lines reduces the scope for missing new types of messages.  If there was one fixed / one new at the same time it would go unnoticed, but that's a bit better already than just a boolean pass/fail.  I'll give that a try.